kuma/javascript/ssr-server.js

```diff
@@ -48,6 +33,28 @@ const MAX_BODY_SIZE = 1024 * 1024 * 5; // 5 megabyte max JSON payload
 // We're using the Express server framework
 const app = express();
 
+// Configure and initialize Sentry if a DSN has been provided.
+if (process.env.SENTRY_DSN) {
+    console.log('Configuring Sentry for the SSR server.');
+    let options = {
+        dsn: process.env.SENTRY_DSN
+    };
+    if (process.env.REVISION_HASH) {
+        options.release = process.env.REVISION_HASH;
+    }
+    if (process.env.SENTRY_ENVIRONMENT) {
+        options.environment = process.env.SENTRY_ENVIRONMENT;
+    }
+    Sentry.init(options);
+
+    // The request handler must be the first middleware on the app
+    app.use(Sentry.Handlers.requestHandler());
+    // The error handler must be before any other error middleware
+    app.use(Sentry.Handlers.errorHandler());
+} else {
+    console.warn('SENTRY_DSN is not available so sentry is not initialized.');
```

Comment:
This was my own personal debugging when I was trying to understand why SENTRY_DSN wasn't enabled. 
After I figured out what was going on I just decided to keep this line because I think it's useful.

Reply:
By the way, to enable Sentry in local development, it's *not* good enough to put it into your `.env`. That'll only help your Python image in docker-compose. It's because the `ssr` service in `docker-compose.yml` doesn't use anything like `dotenv` to read from a `.env` file on disk.